operating/manage-user-access/understanding-multi-tenancy.adoc

```diff
@@ -0,0 +1,97 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="understanding-multi-tenancy"]
+= Understanding multi-tenancy
+include::modules/common-attributes.adoc[]
+:context: understanding-multi-tenancy
+
+toc::[]
+
+[role="_abstract"]
+
+{product-title-short} provides ways to realize multi-tenancy within a Central instance.
+
+This is being done by utilizing role-based access control (RBAC) and access scopes within {product-title-short}.
+
+[role="_additional-resources"]
```

Comment:
I would move the additional resources section to the end unless there is a specific reason that it is here at the beginning.